.github/workflows/test.yml

```diff
@@ -81,7 +81,7 @@ jobs:
         # - without requiring postinstall scripts from dependencies to run
       - name: build
         run: |
-          npm run build --if-present $MODIFIED_WORKSPACES
+          npm run build --if-present
```

Comment:
Is this intentional or was it just an in-between change while building?

Reply:
Something changed that affects how the build results are cached and restored.
Those caches speed up the build process, but now they just cause it to fail often.

I thought I could eliminate the failures by always building everything and still get a small speed bump from only testing that which was changed. But it didn't work.

Currently thinking of removing all caches just to get rid of the failures.
Builds will be slower again, but they will pass reliably.

Reply:
Makes sense!